### DISCUSSION TITLE
Force a certain direction for a layer when generating a core ring?

### DISCUSSION THREAD

**Discussion by donn** [The-OpenROAD-Project/OpenROAD#4438] [Q&A]
We're working with a PDK where the two PDN layers' preferred direction for routing is not the one we want for its part of the core ring, i.e., the layer's preferred direction is vertical but we'd like its part of the core ring to be horizontal. A workaround is modifying the technology LEF for just that step, but is there a way to do that otherwise?

**Comment by maliberty** [comment]
There isn't currently.  I'm curious why it is desirable.
